.github/workflows/go.yaml

```diff
@@ -0,0 +1,64 @@
+name: Go Code
+
+on:
+  push:
+  pull_request:
+    paths:
+    - '*.go'
+    - 'cmd/**'
+    - 'go.mod'
+    - 'go.sum'
+    - 'pkg/**'
+
+jobs:
+  quality:
+    name: Code Quality
```

Comment:
This one is already defined inn `general.yaml`

Reply:
Rename to "Go Code Quality"